openmp/libomptarget/DeviceRTL/CMakeLists.txt

```diff
@@ -304,4 +304,10 @@ add_library(omptarget.devicertl STATIC)
 set_target_properties(omptarget.devicertl PROPERTIES LINKER_LANGUAGE CXX)
 target_link_libraries(omptarget.devicertl PRIVATE omptarget.devicertl.all_objs)
 
+# Install this alongside the LLVM libraries is possible.
+if(NOT OPENMP_STANDALONE_BUILD)
```

Comment:
I think this may break standalone build, especially with the changes in `lit.cfg`.

Reply:
I'm still not sure if this is even an issue, considering we use `add_llvm_library` which already uses this value internally and does the same operation to set its output directory.